Mathlib/Analysis/CStarAlgebra/ContinuousFunctionalCalculus/Order.lean

```diff
@@ -502,6 +504,29 @@ lemma pow_antitone {a : A} (ha₀ : 0 ≤ a := by cfc_tac) (ha₁ : a ≤ 1) :
   peel ha₁ with x hx _
   exact pow_le_pow_of_le_one (spectrum_nonneg_of_nonneg ha₀ hx) (ha₁ x hx) hnm
 
+lemma _root_.nnnorm_cfc_of_monotoneOn [Nontrivial A] (a : A) (f : ℝ≥0 → ℝ≥0)
+    (hf : MonotoneOn f (spectrum ℝ≥0 a)) (hf₂ : ContinuousOn f (spectrum ℝ≥0 a) := by cfc_cont_tac)
+    (ha : 0 ≤ a := by cfc_tac) : ‖cfc f a‖₊ = f ‖a‖₊ := by
```

Comment:
I'm open to putting this lemma and the next somewhere else, I couldn't really think of a better place. Also, I don't think the extra import here is a big deal, but I could be convinced otherwise.

Reply:
I've generalized this in #20882